extension/content.js

```diff
@@ -281,22 +281,23 @@ function addDiffViewWithoutWhitespaceOption(type) {
 	if (urlParams.get('w') === '1') {
 		optionIsSet = true;
 		urlParams.delete('w');
-		optionElementObject.addClass('selected');
 	} else {
 		urlParams.set('w', 1);
 	}
 
-	const optionElementContent = `${optionIsSet ? svgIcon : ''} Ignore whitespace`;
+	const optionElementContent = `${optionIsSet ? svgIcon + ' ' : ''}No Whitespace`;
 	const optionHref = `${window.location.origin + window.location.pathname}?${urlParams.toString() + urlHash}`;
 
-	optionElementObject.html(optionElementContent).attr('href', optionHref).attr('data-hotkey', 'd w').attr('class', 'refined-github-toggle-whitespace');
+	optionElementObject.html(optionElementContent).attr('href', optionHref).attr('data-hotkey', 'd w').attr('class', 'refined-github-toggle-whitespace').attr('title', `Click here to turn whitespace in diffs ${optionIsSet ? 'on' : 'off'}`);
```

Comment:
I'd prefer to build the element using a template string (e.g. https://github.com/sindresorhus/refined-github/blob/master/extension/content.js#L328-L332)